src/frameserver/PointGreyCam.cpp

```diff
@@ -539,7 +541,7 @@ void PointGreyCam<T>::setupAsyncTrigger(int trigger_mode,
         trigger_mode != 1  ||
         trigger_mode != 13 ||
         trigger_mode != 14) {
-        throw (rte("Trigger mode is unsupported."));
+      //throw (rte("Trigger mode is unsupported."));
```

Comment:
Ah, its just a stupid logic error on my part. All the logical ors (`||`) should be and (`&&`). Make that change and I will merge.